### ISSUE TITLE
Issue with openMVG

### ISSUE THREAD

**Issue by stephane-lb** [pyp22/NuxSFM-2.0#2]
Hi Pierre Yves,

Sorry for this second bug report, I  installed opencv and I am now installing openMVG.
Unfortunately, I just get an error : 

```
error: failed to commit transaction (conflicting files)
/usr/local/bin/ui_openMVG_control_points_registration exists in both 'openMVG' and 'openMVG-qt'
Errors occurred, no packages were upgraded.
==> WARNING: Failed to install built package(s).
```

I will look at the PKGBUILD, and I think it is a nice thing to report it.

Kenavo ;-)

Stephane

**Comment by pyp22** [comment]
Hi Stephane, dont' be :) i'm pleased to get some feedback with those new scripts. This a minor error in the script. Bin file is present in both packages. Correcting it and pushing it.

**Comment by stephane-lb** [comment]
Hi Pierre Yves,

It ooks like you already made a fix for this one, on the PCKBUILD for openMVG I found this line 91
```
# Remove ui_openMVG_control_points_registration from here. QT GUI application will have its own package.
```
and then line 122
```
find . -type f -not -name 'ui_openMVG_control_points_registration' | xargs rm
```
Is that something that you forget to commit ?

Have a good evening,

Stephane

**Comment by pyp22** [comment]
Hi Steph, almost yes. The idea is give to UI its own package. Some users (like me xD) are pure command liners and don't need GUI neither GCP UI. It saves spaces as it does not need ton install Xorg and so on. If UI needed it will install automaticly all needed GUI deps.

Pb solved by commit #8785f98

**Comment by pyp22** [comment]
Forget to tell you that you can re clone openMVG and rebuilt it. Or if you kepts src & pkg dirs you can do a makepkg -R -f that will only recompress package.
